### PR TITLE
Create  disable-job-logs.md

### PR DESCRIPTION
**Pull request by nrios14**
New Page for Disabled job logs

### CODE REVIEW

File: user/disable-job-logs.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+|------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| **Access to logs only for write/push users OFF, Access to old logs OFF (default settings)**        | Users with **read access** to the repository can see build job logs for public repo not older than the last X days.                                                                                                                                                                                                                             | Users with **read access** to the repository can see the build job logs not older than the last X days.   |
```
Comment (stan-travis) [inline user/disable-job-logs.md:32]: Instead of "X days" should be "365 days"

File: user/disable-job-logs.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+| **Access to logs only for write/push users ON, Access to old logs OFF**                            | Users with **write access** to the repository will see the build job logs not older than the last X days.                                                                                                                                                                                                                                       | Users with **write access** to the repository will see the build job logs not older than the last X days. |
```
Comment (stan-travis) [inline user/disable-job-logs.md:33]: Instead of "X days" should be "365 days"

Comment (nrios14) [reply user/disable-job-logs.md]: This was updated! Thanks for the feedback

File: user/disable-job-logs.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| **Access to logs only for write/push users ON, Access to old logs ON**                             | Users with **write access** to the repository will see the build job logs, full history                                                                                                                                                                                                                                                     | Users with **write access** to the repository will see the build job logs, full history.              |
+| **Access to logs only for write/push users OFF,  Access to old logs ON(state of today's product)** | Users with **read access** to the repository see all build job logs and full history. In this case, even anonymous Users are treated as Users with read access because everybody can view the public repository in the original VCS. That collaboration pattern must still be available as an option to the repository owners. | Users with **read access** to the repository can see build job logs and full history.                    |
```
Comment (stan-travis) [inline user/disable-job-logs.md:35]: @nrios14 Please remove the sentence: That collaboration pattern must still be available as an option to the repository owners.

Comment (stan-travis) [reply user/disable-job-logs.md]: Obsolete. I already updated.

**Review by michal-at-travisci — CHANGES_REQUESTED** [review]
I think separate file may be insufficient here.

My take would be:
- create snippets for each option, explainign how these do work
- include snippets in the /user/web-ui.md - this is where the user interface question marks in Repository Settings are leading
- if deemed necessary, re-include snippets in separate document proposed in this PR

**Review by GbArc — APPROVED** [review]

**Review by hhjmmmmmmmm — APPROVED** [review]

**Review by hhjmmmmmmmm — APPROVED** [review]

**Review by hhjmmmmmmmm — APPROVED** [review]

**Review by hhjmmmmmmmm — APPROVED** [review]

**Review by hhjmmmmmmmm — APPROVED** [review]

**Pull request merged**